### PR TITLE
[17.0][MIG] hr_employee_partner_external: Migration to 17.0

### PR DESCRIPTION
**Pull request by BernatObrador**
Module migrated to version 17.0

cc https://github.com/APSL 158258

@miquelalzanillas @lbarry-apsl @mpascuall @peluko00 @javierobcn @ppyczko please review

### CODE REVIEW

**Comment by ppyczko** [comment]
LGTM, tested in runboat

**Review by mpascuall — APPROVED** [review]
LGTM

**Review by ppyczko — APPROVED** [review]
LGTM, code review and tested in runboat

**Review by peluko00 — APPROVED** [review]
LGTM

**Comment by BernatObrador** [comment]
@pedrobaeza is this ready to merge?

File: hr_employee_partner_external/README.rst [hunk 1]
```diff
@@ -80,6 +80,10 @@ Contributors
 ------------
 
 -  Stefano Consolaro <stefano.consolaro@mymage.it>
+-  `APSL-Nagarro <https://apsl.tech>`__:
+
+   -  Bernat Obrador <bobrador@apsl.net>
```
Comment (pedrobaeza) [inline hr_employee_partner_external/README.rst:85]: 2 new contributors for this simple migration seems too much.

Comment (BernatObrador) [reply hr_employee_partner_external/README.rst:85]: Done.

**Review by pedrobaeza — APPROVED** [review]
/ocabot migration hr_employee_partner_external
/ocabot merge nobump

**Pull request merged**